modules/test/playwright/fixtures/usersAndOrganizationsPagesTest.ts

```diff
@@ -41,6 +47,9 @@ const usersAndOrganizationsPagesTest = test.extend<{
 	assignUsersPage: async ({page}, use) => {
 		await use(new AssignUsersPage(page));
 	},
+	documentLibraryPage: async ({page}, use) => {
+		await use(new DocumentLibraryPage(page));
+	},
```

Comment:
this will be only the new one (see comment below) and in the test we have to import also the "original one"